lib/Configuration/OptionsStorage.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * @author Juan Pablo Villafáñez <jvillafanez@solidgear.es>
+ *
+ * @copyright Copyright (c) 2018, ownCloud GmbH
+ * @license AGPL-3.0
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\Notifications\Configuration;
+
+use OCP\IConfig;
+
+class OptionsStorage {
```

Comment:
PHPDoc, what's an "OptionsStorage" ? An external storage to store options ?

OptionsService ?

Reply:
I'll add a PHPDoc to clarify. The intention is to hide where the configuration options are stored (currently in the preferences, but we might need to move then to another table if this grows too much), as well as to provide an easier way to access to those options.